apps/demo/src/Create.tsx

```diff
@@ -17,9 +17,9 @@ import {
 } from "@openfun/cunningham-react";
 import { faker } from "@faker-js/faker";
 import { Character, database, randomDates } from "./Character";
-import { Page, PageProps } from "./App";
+import { PageProps } from "./App";
 
-export const Create = ({ changePage }: PageProps) => {
+export const Create = ({ modal }: PageProps) => {
```

Comment:
I wonder if we can refactor this component in order to make it extend the Modal component to directly pass `useModal()` data to it from the parent component 🤔